test/test_pipe_baseline.py

```diff
@@ -1,21 +1,73 @@
 import unittest
+import resonance.pipe as pipe
+import resonance.si as si
+import resonance.db as db
 import resonance.run as run
+import numpy as np
 
 
 class TestPipeBaseline(unittest.TestCase):
-    def check(self, si, blocks, expected):
-        code = '''
-from resonance import *
-in = input(1)
-out = pipe.baseline(in)
-createOutput(out, 'out')
-'''
+    def check(self, si, blocks, expected, beginOffset, endOffset):
+
+        code = "from resonance import *"\
+               "inputData = input(0)"\
+               "out = pipe.baseline(inputData, {0}, {1})"\
+               "createOutput(out, 'out')"\
+               .format(beginOffset, endOffset)
 
         online = run.online([si], blocks, code)
         offline = run.offline([si], blocks, code)
 
         self.assertEqual(online, expected)
         self.assertEqual(online, offline)
 
-    def test_spatial(self):
-        pass
+    def test_baseline(self):
+        channels = 2
+        samples = 20
+        time = 3
+
+        w_si = si.Window(channels, samples, 250)
```

Comment:
Single window in all tests, can you pass several windows?
In fact them could be the same, only amount matters